lib/CfClient.dart

```diff
@@ -131,18 +132,19 @@ class CfClient {
 
   /// Initializes the SDK client with provided API key, configuration and target. Returns information if
   /// initialization succeeded or not
-  Future<InitializationResult> initialize(
-      String apiKey, CfConfiguration configuration, CfTarget target) async {
+  Future<InitializationResult> initialize(String apiKey,
+      CfConfiguration configuration, CfTarget target) async {
     _hostChannel.setMethodCallHandler(_hostChannelHandler);
 
+    // TODO - we need a try-catch here on PlatformException in order to retry.
```

Comment:
We can wrap this code in a try-catch for future enhancement when we add retry's to the SDK.

For now, this will just mean any errors returned from the method channel are logged.